config/widgets/settings/generalOptionsTab.json

```
@@ -106,6 +110,12 @@
 					"help": "vcmi.systemOptions.longTouchButton",
 					"callback": "setLongTouchDuration",
 					"created" : "touchscreen"
+				},
+				{
+					"name": "hapticFeedbackCheckbox",
+					"help": "vcmi.systemOptions.hapticFeedbackButton",
+					"callback": "hapticFeedbackChanged",
+					"created" : "touchscreen"
```

For consistency - use same condition as in label creation
```suggestion
					"created" : "mobile"
```

---

unrelated but how do we handle stuff like microsoft surface tablets? :P

---

Same as SDL :)

Generally, touch input should work on any system as long as it is supported by SDL. I see some conflicting info on SDL touch input on Windows, so not sure if it is supported.

Linux-based systems (non-Android) should work, e.g. Steam Deck, but obviously not tested.